README.md

```diff
@@ -40,6 +40,8 @@ tctl --ns test-namespace namespace register -rd 1
 
 Get started building Workflows with a [Go sample](https://github.com/temporalio/samples-go), [Java sample](https://github.com/temporalio/samples-java), or write your own using one of the [SDKs](https://docs.temporal.io/docs/sdks-introduction).
 
+> Note for running this `docker-compose` on M1 Apple Silicon: you may run into a `no matching manifest for linux/arm64/v8 in the manifest list entries` error. To resolve this, [specify the platform](https://stackoverflow.com/questions/65456814/docker-apple-silicon-m1-preview-mysql-no-matching-manifest-for-linux-arm64-v8) for the Cassandra container.
```

Comment:
Why not just to add:
```
platform: linux/x86_64
```
to `docker-compose*.yaml` files? There should be no harm doing it.

Reply:
actually as I found out we do need the ARM build, so i need to change this run note altogether...